### PR TITLE
MEPTS-980 | RESUMO MENSAL PREP - MISAU

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DatimCodeDatasetDefinition.java [hunk 1]
```diff
@@ -11,24 +10,40 @@
...
 
+import java.util.List;
+
 /**
  * Simple dataset of location_attribute properties. Actually supported:
  * location_attribute_type.name, location.name and location_attribute.value_reference => Datim Code
  */
 public class DatimCodeDatasetDefinition extends EvaluatableDataSetDefinition {
 
+  int typeId;
```
Comment (eliazmuchave) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DatimCodeDatasetDefinition.java]: Should add access modifiers for both fields

**Review by eliazmuchave — APPROVED** [review]
Looks good. Can be merged

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DatimCodeDatasetDefinition.java [hunk 2]
```diff
@@ -11,24 +10,40 @@
...
  * Simple dataset of location_attribute properties. Actually supported:
  * location_attribute_type.name, location.name and location_attribute.value_reference => Datim Code
  */
 public class DatimCodeDatasetDefinition extends EvaluatableDataSetDefinition {
 
+  protected int typeId;
+
+  protected String columnName;
+
+  protected String getQuery(int typeId, String columnName) {
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DatimCodeDatasetDefinition.java:25]: are those variable declared protected meant to be used somewhere else in the package? if NOT please declare them as private

Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DatimCodeDatasetDefinition.java:25]: Yes, they are being used in another file on the same package. Pls see sismaCodeDatasetDefinition.java

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DatimCodeDatasetDefinition.java:25]: Ok

**Review by ningosi — APPROVED** [review]

**Pull request merged**